php-checks/src/main/java/org/sonar/php/checks/utils/PhpUnitCheck.java

```diff
@@ -31,6 +37,202 @@
 
   private boolean isPhpUnitTestCase = false;
   private boolean isPhpUnitTestMethod = false;
+  private static final Map<String, Assertion> ASSERTION = assertions();
+
+  private static Map<String, Assertion> assertions() {
+    return Stream.of(
+      new Assertion("assertArrayHasKey"),
+      new Assertion("assertArraySubset"),
+      new Assertion("assertArrayNotHasKey"),
+      new Assertion("assertContains"),
+      new Assertion("assertContainsEquals"),
+      new Assertion("assertAttributeContains"),
+      new Assertion("assertNotContains"),
+      new Assertion("assertNotContainsEquals"),
+      new Assertion("assertAttributeNotContains"),
+      new Assertion("assertContainsOnly"),
+      new Assertion("assertContainsOnlyInstancesOf"),
+      new Assertion("assertAttributeContainsOnly"),
+      new Assertion("assertNotContainsOnly"),
+      new Assertion("assertAttributeNotContainsOnly"),
+      new Assertion("assertCount", true),
+      new Assertion("assertAttributeCount", true),
+      new Assertion("assertNotCount", true),
+      new Assertion("assertAttributeNotCount", true),
+      new Assertion("assertEquals", true),
+      new Assertion("assertEqualsCanonicalizing", true),
+      new Assertion("assertEqualsIgnoringCase", true),
+      new Assertion("assertEqualsWithDelta", true),
+      new Assertion("assertAttributeEquals", true),
```

Comment:
This and assertAttributeSame results in FPs in regards to the implemented rule in the ITS. The second argument is here always the attribute name. Also check the other assertAttribute* functions. Maybe it makes more sense to remove the hasExpectedValue from them